velox/docs/functions/presto/conversion.rst

```diff
@@ -489,6 +489,56 @@ Valid examples
   SELECT cast(cast(0.123 as DECIMAL(3, 3)) as varchar); -- '0.123'
   SELECT cast(cast(1 as DECIMAL(6, 2)) as varchar); -- '1.00'
 
+From Floating-Point Types
+^^^^^^^^^^^^^^^^^^^^^^^^^
+By default, casting a real or double to string returns standard notation if the magnitude of input value is greater than
+or equal to 10 :superscript:`-3` but less than 10 :superscript:`7`, and returns scientific notation otherwise. For some
+rare cases the result precision differs from Presto by 1 digit.
```

Comment:
> For some rare cases the result precision differs from Presto by 1 digit.

I think this may be too detailed. It's OK to leave it out IMO.
By the way, why "by 1 digit"? Is the difference somehow bounded?

Reply:
Sure, I agree with leaving the detail out, as it might create wondering.

Regarding "1 digit", we see the majority of cases has a max of 17 significant digits for double and 8 digits for float, as you know as it's determined by machine representation. But Presto sometime has one more least significant digit and it is inaccurate (or accurate).